compressor/storage.py

```diff
@@ -4,12 +4,22 @@
 import time
 from urllib.parse import urljoin
 
-from django.core.files.storage import FileSystemStorage, get_storage_class
+import django
+from django.core.files.storage import FileSystemStorage
 from django.utils.functional import LazyObject, SimpleLazyObject
 
 from compressor.conf import settings
 
 
+def _get_storage_class(backend):
+    if django.VERSION[0] > 4 or (django.VERSION[0] == 4 and django.VERSION[1] > 1):
+        from django.core.files.storage import storages
+        return storages.create_storage({"BACKEND": backend})
+    else:
+        from django.core.files.storage import get_storage_class
+        return get_storage_class(backend)()
```

Comment:
I wonder if we should be using `storages[backend]` here (and likely `override_settings`) 🤔

Reply:
Would that work? I think `storages` was only introduced in Django 4.2?

Reply:
I think currently `storages[backend]` wouldn't have anything in it?

Are we saying that users should be declaring a backend with `compressor.storage.CompressorFileStorage` for us to use, so we don't keep creating a new instance?

Reply:
Well, the whole `storages` thing is new so... 😜

I think if we were designing from scratch, we'd likely say something like, "If you declare a backend in `STORAGES` we'll use that, otherwise we'll automatically create one for you". 

So we'd need a setting for the `backend` name, and then we could look for that on first access. 

That's just a first thought. Open to suggestions.